src/main/webapp/release.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <title>Release View</title>
+      <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+     <div class="jumbotron">
+        <h1>SDK Release</h1>      
+    </div>  
+
```

Comment:
It would be nice if you could take inputs from some JSON file with the release data, so you won't have to re-do this whole section later when you are trying to display data based on the API calls.
The other option is to return this dummy data from the API endpoints once they are created. That would be ok too.

Reply:
I will implement getting dummy data from API endpoints and I wanted to have all the pieces of the prototype together.